ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/controller/TransactionController.java

```diff
@@ -25,19 +25,20 @@ public class TransactionController {
   @PostMapping(
       value = API_PATH + "/local/{transportDocumentHash}",
       consumes = MediaType.APPLICATION_JSON_VALUE,
-      produces = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity<UUID> createLocalTransaction(
+      produces = MediaType.TEXT_PLAIN_VALUE)
+  public ResponseEntity<String> createLocalTransaction(
```

Comment:
So... DCSA Design Principles state that endpoints _MUST_ provide JSON (and _MAY_ provide alternative content).

I am of two minds on this in this case - but wanted to make sure this is a deliberate break from the design principles (because then it implies that we change MUST to a less strong word - which is probably healthy for other reasons).

Reply:
Alternative, you can also call "unofficial, DCSA Design Principles do not apply here"

Reply:
I think this being an unofficial endpoint this is fine